python/hail/tests/test_methods.py

```diff
@@ -706,6 +706,66 @@ def test_sample_qc(self):
         dataset = self.get_dataset()
         dataset = hl.sample_qc(dataset)
 
+    def test_variant_qc(self):
+        data = [
+            {'v': '1:1:A:T', 's': '1', 'GT': hl.Call([0, 0]), 'GQ': 10, 'DP': 0},
+            {'v': '1:1:A:T', 's': '2', 'GT': hl.Call([1, 1]), 'GQ': 10, 'DP': 5},
+            {'v': '1:1:A:T', 's': '3', 'GT': hl.Call([0, 1]), 'GQ': 11, 'DP': 100},
+            {'v': '1:1:A:T', 's': '4', 'GT': None, 'GQ': None, 'DP': 100},
+            {'v': '1:2:A:T,C', 's': '1', 'GT': hl.Call([1, 2]), 'GQ': 10, 'DP': 5},
+            {'v': '1:2:A:T,C', 's': '2', 'GT': hl.Call([2, 2]), 'GQ': 10, 'DP': 5},
+            {'v': '1:2:A:T,C', 's': '3', 'GT': hl.Call([0, 1]), 'GQ': 10, 'DP': 5},
+            {'v': '1:2:A:T,C', 's': '4', 'GT': hl.Call([1, 1]), 'GQ': 10, 'DP': 5},
+        ]
+
```

Comment:
Was variant qc never tested? Or are there Scala tests you can delete?

Reply:
I can't delete the scala tests yet because there are things that depend on it :(

Reply:
but there were no python tests 😱